ROADMAP.md

```diff
@@ -10,75 +10,84 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
-* Durability of state (local storage support, config can be regenerated) [#1011](https://github.com/rook/rook/issues/1011) [#592](https://github.com/rook/rook/issues/592)
-* Integration testing improvements
-  * Update promotion and release channels to align with storage provider specific statuses [#1885](https://github.com/rook/rook/issues/1885)
-  * Refactor test framework and helpers to support multiple storage providers [#1788](https://github.com/rook/rook/issues/1788)
-  * Isolate and parallelize storage provider testing [#1218](https://github.com/rook/rook/issues/1218)
-  * Longhaul testing pipeline [#1847](https://github.com/rook/rook/issues/1847)
-* Support for dynamic provisioning of new storage types
-  * Dynamic bucket provisioning [#1705](https://github.com/rook/rook/issues/1705)
-  * Dynamic database provisioning [1704](https://github.com/rook/rook/issues/1704)
 * Cassandra
   * Admission webhook to reduce user error [#2363](https://github.com/rook/rook/issues/2363)
   * Integrate prometheus monitoring [#2530](https://github.com/rook/rook/issues/2530)
-  * Integrate with Spotify Reaper to provide repairs [#2531](https://github.com/rook/rook/issues/2531)
-  * Dealing with loss of persistence: leverage cassandra's mechanisms to detect when data has been lost and stream it from other nodes [#2533](https://github.com/rook/rook/issues/2533)
+  * Enable automated repairs [#2531](https://github.com/rook/rook/issues/2531)
+  * Handle loss of persistent local data [#2533](https://github.com/rook/rook/issues/2533)
   * Minor version upgrades [#2532](https://github.com/rook/rook/issues/2532)
 * Ceph
-  * Ceph-CSI plug-in integration [#1385](https://github.com/rook/rook/issues/1385)
-    * Resizing volumes [#1169](https://github.com/rook/rook/issues/1169)
-  * Improved data placement and pool configuration (CRUSH maps) [#560](https://github.com/rook/rook/issues/560)
-  * OSDs
-    * Run on arbitrary PVs (local storage) as an alternative to host path [#919](https://github.com/rook/rook/issues/919)
-    * Detect new nodes for OSD configuration [#2208](https://github.com/rook/rook/issues/2208)
-    * Disk management (adding, removing, and replacing disks) [#1435](https://github.com/rook/rook/issues/1435)
-  * File
-    * NFS Ganesha CRD [#1799](https://github.com/rook/rook/issues/1799)
-    * Dynamic Volume Provisioning for CephFS [#1125](https://github.com/rook/rook/issues/1125)
-  * Object
-    * Multi-site configuration [#1584](https://github.com/rook/rook/issues/1584)
-  * Mgr and plugins
-    * Placement group balancer support (enable the mgr module)
+  * Add support for Ceph Nautilus
+  * Beta release of CSI plug-in [#1385](https://github.com/rook/rook/issues/1385)
+    * Dynamic volume resizing [#1169](https://github.com/rook/rook/issues/1169)
+  * Automatically detect new nodes [#2208](https://github.com/rook/rook/issues/2208)
+  * Disk management (adding, removing, and replacing disks) [#1435](https://github.com/rook/rook/issues/1435)
```

Comment:
is adding/removing disks still likely for 1.0?

Reply:
@noahdesu ?

Reply:
Yeh. I'll have a preliminary PR up soon. It's working, but I think some of the rough edges will attract some comments that shouldn't be too bad to smooth out.